app/Support/Version.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Support;
+
+use App\Enums\VersionEnum;
+use Illuminate\Support\Facades\Route;
+
+class Version
+{
+    public static function current(): string
+    {
+        $route = Route::getRoutes()->match(request());
+
+        return $route->parameters()['version'] ?? str_replace('/v', '', $route->getPrefix());
+    }
+
+    public static function is(VersionEnum $checkedVersion): bool
+    {
+        return $checkedVersion->value === self::current();
```

Comment:
please use `version_compare` helper for all comparation operations in this class

Reply:
Done.